debezium-storage/debezium-storage-kafka/src/main/java/io/debezium/storage/kafka/history/KafkaSchemaHistory.java

```diff
@@ -341,6 +351,8 @@ protected void recoverRecords(Consumer<HistoryRecord> records) {
                             lastProcessedOffset = record.offset();
                             ++numRecordsProcessed;
                         }
+                        LOGGER.info("Sleeping for {} seconds !", config.getLong(SCHEMA_HISTORY_RECOVERY_DELAY_MS)/1000);
+                        Thread.sleep(config.getLong(SCHEMA_HISTORY_RECOVERY_DELAY_MS));
```

Comment:
nit: Either remove the log statement or keep it at debug/trace level.

We are adding a sleep call for just for the testing. This is in the production code-path and would be better to avoid. But I am not able to think of any other better approach. Let's revisit this together sometime. WDYT?

Reply:
Removing this, as this was required only for manual testing.